pgcli/pgexecute.py

```diff
@@ -226,12 +220,22 @@ def _json_typecaster(self, json_data):
         else:
             return json_data
 
-    def run(self, statement, pgspecial=None, on_error=ON_ERROR_RESUME):
+    def run(self, statement, pgspecial=None, exception_formatter=None,
```

Comment:
The default argument for `exception_formatter` should be `repr`. The exception_formatter is an optional function that does coloring, when one is not provided we should do the bare minimum which is convert the exception to it's default string representation.